src/components/Sheet/Sheet.scss

```diff
@@ -82,6 +90,10 @@ $block: '.#{variables.$ns}sheet';
         &_without-scroll {
             overflow: hidden;
         }
+
+        @media (prefers-reduced-motion: reduce) {
+            transition-duration: 1ms;
+        }
```

Comment:
I guess that block should start at line 89, because we override `&__sheet-scroll-container` styles

Reply:
`&_without-scroll` is also a mod for `sheet-scroll-container`. So there is no difference in order, both rules are applied to the same block, and not interfere with each other